test/data_lake/test_data_lake.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019-present MongoDB, Inc.
+#
```

Comment:
Does this need be to it's own `data_lake` module or could we move these tests into `test/test_data_lake.py`?